src/typescript/frontend/src/components/OrderbookTable.tsx

```diff
@@ -40,13 +40,25 @@ const Row: React.FC<{
     baseCoinDecimals: BigNumber(marketData.base?.decimals || 0),
   });
 
+  const barPercentage = (level.size * 100) / highestSize;
+  const barColor =
+    type === "bid" ? "rgba(110, 213, 163, 30%)" : "rgba(213, 110, 110, 30%)";
+
   return (
     <div
       className="relative flex h-6 cursor-pointer items-center justify-between py-[1px] hover:ring-1 hover:ring-neutral-600"
       onClick={() => {
         setType(type === "ask" ? "buy" : "sell");
         setPrice(price.toString());
       }}
+      style={{
+        background: `linear-gradient(
+          to left,
+          ${barColor},
+          ${barColor} ${barPercentage}%,
+          transparent ${barPercentage}%
+        )`,
+      }}
```

Comment:
jw is this better practice and setting widths manually like i did before an antipattern?
regardless everything looks great

Reply:
oh NVM didn't see ur comment in slack. diregard!